airflow/models/dag.py

```diff
@@ -2656,11 +2655,6 @@ def deactivate_deleted_dags(cls, alive_dag_filelocs: List[str], session=None):
             if dag_model.fileloc is not None:
                 if correct_maybe_zipped(dag_model.fileloc) not in alive_dag_filelocs:
                     dag_model.is_active = False
-                else:
-                    # If is_active is set as False and the DAG File still exists
-                    # Change is_active=True
-                    if not dag_model.is_active:
-                        dag_model.is_active = True
```

Comment:
Why this change? what will re-activate the DAG if it is readded

Reply:
> Why this change?

With this code in place, any time a DAG is no longer present in a file it will be continually reactivated, so this code needs to be removed in order to fix this issue. 

I could not find a good explanation for why this was added in the first place ([here's the PR which introduced it](https://github.com/apache/airflow/pull/5743/files)), so I think its safe to remove. 

> what will re-activate the DAG if it is readded

I believe that the next time a DAG is found in a file, it will be marked as `active`. 

https://github.com/apache/airflow/blob/c73004d0cd33d76b82b172078f572e8d4eecab39/airflow/models/dag.py#L2406

Reply:
Yup, make sense